### DISCUSSION TITLE
Urgent Request for Student Plan - Review of Documents

### DISCUSSION THREAD

**Discussion by felpsg** [community/community#63193] [GitHub Education]
### Hi everyone,

Dear GitHub Support,

 I am a student at Faculdade de Americana. I am writing to express my concern regarding my application for the GitHub Student Plan. Despite following all provided instructions and submitting multiple authentic documents as proof, my application has been denied more than four times.

The submitted documents include:

Student ID card with photograph and relevant details.
Official letterhead document from my institution containing all necessary information.
Other authentic proofs of my enrollment.
Unfortunately, my institution does not provide student emails, as studying is conducted through a specific platform. This has become a barrier in obtaining the student plan, which is vital to my development as a full-stack developer.

I understand that verification procedures are essential in ensuring the authenticity of applications, but I assure you that all provided documents are legitimate and valid.

I kindly request that my application be reassessed, and I am willing to provide any additional information or documentation that may be required.

Thank you in advance for your attention to this matter, and I eagerly await a positive response.

Sincerely,

Felipe Gustavo Alves Gonçalves

**Comment by humairshoukat** [comment]
Try submitting you're student ID card picture fro your phone. Try taking the picture while submitting the document.

**Reply by felpsg** [reply]
I have already done this; I have already sent my identification card, already sent an official document from the university attesting that I am a student with the rector's signature, and they reject all of it. I made it clear that my college does not use a student email because we use the platform and have an email integrated with it, but none of that helps. I even sent the website to verify my student card with the code written on the card to validate the authenticity; this is frustrating!

**Comment by humairshoukat** [comment]
Contact a GitHub Campus Expert. He/She can raise an internal issue on your behalf. Find your nearest one here https://education.github.com/experts
